### PR TITLE
Changes in linux intro presentation

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: docs/course_materials/linux_intro.html [hunk 1]
```diff
@@ -342,59 +518,47 @@ <h3><b>tail</b></h3>
...
         </section>
       </section>
       <section>
         <section>
-          <h3><b>clear</b></h3>
+          <h3><b>Exercise</b></h3>
           <ul>
-            <li>Clear the terminal screen</li>
+            <li>Open a terminal </li>
+            <li>Navigate to /test/tmp in your home directory</li>
```
Comment (sleviim) [inline docs/course_materials/linux_intro.html:536]: /test/tmp -> test/tmp

File: docs/course_materials/linux_intro.html [hunk 2]
```diff
@@ -39,6 +39,13 @@
         <h1 class="slide_main_title"><b>Linux Introduction</b></h1>
       </section>
       <section>
+        <section>
+          <h3><b>Terminal </b></h3>
```
Comment (sleviim) [inline docs/course_materials/linux_intro.html:43]: trailing space at the end

File: docs/course_materials/linux_intro.html [hunk 3]
```diff
@@ -196,21 +351,19 @@ <h3><b>tree</b></h3>
...
-      </code></pre>
-          <ul>
-            <li>Remove a directory and all parent directories</li>
+          <h3><b>Exercise</b></h3>
+          <ul>
+            <li>Open a terminal </li>
+            <li>Navigate to your home directory</li>
+            <li>create a directory called test</li>
+            <li>Navigate to test directory</li>
+            <li>create a directory called tmp in test directory</li>
```
[the remark this answers is not in the document]
Comment (shiramax) [reply docs/course_materials/linux_intro.html:360]: It's might be redundant but I prefer to repeat myself twice for it to be more clear.

File: docs/course_materials/linux_intro.html [hunk 4]
```diff
@@ -296,6 +452,26 @@ <h3><b>rm</b></h3>
   </code></pre>
         </section>
       </section>
+      <section>
+        <section>
+          <h3><b>rmdir</b></h3>
```
[the remark this answers is not in the document]
Comment (shiramax) [reply docs/course_materials/linux_intro.html:457]: Yes, but I've decided to just move it to a different section

**Comment by shiramax** [comment]
@sleviim I'm closing this in favor https://github.com/RedHat-Israel/ROSE/pull/338
I've addressed all your comments. thanks !

**Pull request closed without merging**